app/scripts/views/complete_reset_password.js

```diff
@@ -61,10 +52,13 @@ function (Cocktail, BaseView, FormView, Template, PasswordMixin,
       }
 
       var self = this;
-      return this.fxaClient.isPasswordResetComplete(this.token)
+      var verificationInfo = this._verificationInfo;
+      var token = verificationInfo.get('token');
+      return this.fxaClient.isPasswordResetComplete(token)
         .then(function (isComplete) {
-          self._isLinkExpired = isComplete;
           if (isComplete) {
+            verificationInfo.setValidationError(
+                AuthErrors.toError('EXPIRED_VERIFICATION_LINK'));
             self.logEvent('complete_reset_password.link_expired');
```

Comment:
If we are going to go through the trouble of adding a new Error type, might as well do a `self.logError` with the new error here.